src/NuGetTrends.Api/PackageController.cs

```diff
@@ -22,7 +22,7 @@ public class PackageController : ControllerBase
         public async Task<ActionResult<IEnumerable<object>>> Search([FromQuery] string q, CancellationToken cancellationToken)
             => await _context.PackageDownloads
                 .AsNoTracking()
-                .Where(p => p.PackageIdLowered.Contains(q.ToLower(CultureInfo.InvariantCulture)))
+                .Where(p => p.PackageIdLowered != null && p.PackageIdLowered.Contains(q.ToLower(CultureInfo.InvariantCulture)))
```

Comment:
`PackageIdLowered` cannot be null here, right? So maybe use the null-forgiving operator?

Reply:
Good point. Would need to make sure none is null in the DB and we can't get null go through

Reply:
It's marked as Required in the `DbContext` configuration so I think it should be okay.

Reply:
Honestly will make a new PR for that.

Reply:
Sounds good.